lib/deploy/build-riffraff-bundle.js

```diff
@@ -34,25 +34,21 @@ const { sites, root, dist, target } = require('../../config');
 const copyCfn = () => {
     log(' - copying cloudformation config');
     return cpy(
-        ['frontend/cloudformation.yml'],
+        [`${siteName}/cloudformation.yml`],
         path.resolve(target, 'frontend-cfn'),
```

Comment:
There are a couple places where "frontend " is hardcoded, and others where `siteName` is used. I wonder whether we just want to either replace all references of "frontend" with `siteName`? Or replace references to `siteName` with "frontend"?

Reply:
I'd be inclined to go with the former: replace all references of "frontend" with `siteName`. eg. `path.resolve(target, `${siteName}-cfn`)`.